src/macaron/__main__.py

```diff
@@ -167,10 +167,18 @@ def main() -> None:
         "--policy",
         required=False,
         default=[],
-        help=("The path to a policy yaml file or directory."),
+        help="The path to a policy yaml file or directory.",
         action="append",
     )
 
+    main_parser.add_argument(
```

Comment:
Looks like the list of repositories can be provided both through args and `defaults.ini`. That would require consolidating the values if provided in both ways. To avoid these extra checking, I think it's better to just read it through `defaults.ini`.